_includes/v21.1/orchestration/start-cockroachdb-secure.md

```diff
@@ -78,7 +78,7 @@ The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so
         pvc-9e4f57f0-fb2e-11e9-a65c-42010a8e0fca   100Gi      RWO            Delete           Bound    default/datadir-cockroachdb-2   standard                51m
         ~~~
 
-    1. Run `cockroach init` on one of the pods to complete the node startup process and have them join together as a cluster:
+    1. Run `cockroach init` on one of the pods to complete the node startup process and have them combine as a cluster:
```

Comment:
Ah, "combine" really isn't appropriate here I think. Maybe we remove this Vale rule entirely for now and revisit separately? I have a few ideas for rewriting this step, but none stand out as ideal at present. Key issue: "Combine" is the keyword for the action at hand, and is not present anywhere around `--join` or the "Start a Local Cluster" docs. I think we actually lose comprehension by dropping the common concept: "join".

Reply:
Reverted to master and disabled the `vale.Redundancy` rule for now.

We can take any of the style rules from the Vale source code [here](https://github.com/errata-ai/vale/tree/v2/styles) and adapt them to our specific needs.